nw-firestore-schema.ts

```diff
@@ -48,6 +48,13 @@ interface Quicklink {
   common: boolean;
 }
 
+interface Prize {
+  place: number | boolean; // number if a main prize, false if sponsor prize
+  sponsor: string;
+  title: string;
+  content: string; // | delimited 
```

Comment:
Instead of delimiting can we make this a 'rich' string (with line breaks etc)?

Reply:
n00b question - 
do you mean to delimit with the new line character instead of the pipe character, on the client side? ie. i should update the live site to delimit with \n instead of |, and also update this comment 

in the livesite, the delimited substrings each get put into <LI> html elements to make a <UL>

Reply:
previously we've used text areas and markdown to style anything more complicated than a single line of text. Do you think this is something we could use here?

its worth noting that saving a string 
```
- item 1
- item 2 
```
in firebase keeps it's line breaks

Reply:
no, please correct me if I'm wrong tho 🙏 
currently the front end takes the |-delimited string, parses it into an array of substrings, then creates a styled list item for each item in that array. 

if we were to use a rich string or markdown, we still need to parse it to create the styled list items. If this is what you mean, I can make the change in the front end as well, lmk!

Reply:
Sounds like you just want an array. They're natively supported by firebase.

Reply:
oh sorry I didn't see this. yes that is exactly what I meant, I thought it would be better to use the same method of parsing here as in other areas of the livesite

Reply:
two reasons, consistency with the announcement section of livesite, and easier to implement on the cms (dealing with a markdown string is much easier than an array or creating a delimited string)